include/arch/x86/linker.ld

```diff
@@ -136,6 +136,17 @@ SECTIONS
 
 	__data_ram_end = .;
 
+#ifdef CONFIG_FIRMWARE_VERSION_STRING
+	SECTION_PROLOGUE(_VERSION_SECTION_NAME, (OPTIONAL),)
+	{
+	. = ALIGN(0x40033FC0);
```

Comment:
Give the address a name, put it in linker-defs with all the other defs.
Surely you're aware by now that this is something we need to do?

Just always assume that you need to do this. We really don't need to spend time on trivial things like this in code review.